modules/upgrade/pages/migrate/kubernetes/strimzi.adoc

```diff
@@ -0,0 +1,351 @@
+--
+.Strimzi
+[source,yaml]
+----
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: my-topic <1>
+  labels:
+    strimzi.io/cluster: my-kafka-cluster <2>
+spec:
+  partitions: 3 <3>
+  replicas: 3 <4>
+----
+
+.Redpanda
+[source,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Topic
+metadata:
+  name: my-topic <1>
+spec:
+  kafkaApiSpec: <2>
+    brokers:
+      - "redpanda-0.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-1.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-2.redpanda.<namespace>.svc.cluster.local:9093"
+    tls:
+      caCertSecretRef:
+        name: "redpanda-default-cert"
+        key: "ca.crt"
+  partitions: 3 <3>
+  replicationFactor: 3 <4>
+----
+--
+
+<1> *Topic name*: Both configurations identify the Kafka topic by the same name, ensuring consistency across migration.
+
+<2> *Cluster reference*: Strimzi uses labels to link the topic to the specific Kafka cluster, whereas Redpanda uses a `kafkaApiSpec` block, explicitly defining the brokers and security settings.
+
+<3> *Partitions*: Both platforms maintain the same number of partitions for the topic, facilitating a direct translation of partition configuration.
+
+<4> *Replicas*: The number of replicas is set to 3 in both cases, ensuring high availability and data redundancy during and after migration.
+
+=== Migrate the KafkaUser resource
+
+The Redpanda Operator does not support a custom resource for Kafka users. For details on user authentication in Redpanda, see xref:manage:kubernetes/security/authentication/k-authentication.adoc[].
+
+==== Migrate the KafkaConnect resource
+
+The Redpanda Operator does not support a custom resource to define Kafka Connect deployments. Redpanda provides support for Kafka Connect through a separate Connectors Helm chart.
+
+See xref:deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc[].
+
+==== Migrate the KafkaBridge resource
+
+Redpanda includes a built-in HTTP proxy on each broker, enabling direct HTTP-based interactions without the need for a separate bridge component. For details, see xref:develop:http-proxy.adoc[]. For the API reference, see xref:api:ROOT:pandaproxy-rest.adoc[].
+
+==== Migrate the KafkaMirrorMaker2 resource
+
+The Redpanda Operator does not support a custom resource for MirrorMaker2. Redpanda offers a separate Helm chart that includes Kafka Connect and MirrorMaker2. For information on how to use MirrorMaker2 with Redpanda in Kubernetes:
+
+. xref:deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc[].
+. xref:manage:kubernetes/k-manage-connectors.adoc[].
+
+==== Migrate the KafkaNodePool resource
+
+In Strimzi, KafkaNodePool resources represent different groups of Kafka nodes (brokers). All brokers from a given pool share the same configuration. But their configuration might differ from brokers belonging to other node pools.
+
+Redpanda Operator does not have an equivalent for KafkaNodePool resources.
```

Comment:
I'm a bit confused. If RP does not have an analogous feature/function to KafkaNodePool resources, why are we telling users to migrate it? What, exactly, are they migrating it to?

Reply:
Since we don't support it, users would need to take into account that all brokers must be configured the same. I've clarified this point 😄